.ci/magician/cmd/vcr_merge_test.go

```diff
@@ -0,0 +1,96 @@
+package cmd
+
+import (
+	"magician/github"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestExecVCRMerge(t *testing.T) {
```

Comment:
After the RunE switch could you add some tests for error cases here?

Reply:
Added a test specific for RunE, not sure that's the way. It does not check argument number. 
It is strange to test a subcommand, since if calling Execute on the subcommand, it will find the parent to execute first. 
Or we move all the logic within RunE into the execVcrMerge function itself, then only testing that function.

Reply:
sorry for being unclear - what I meant is that after converting to RunE it should be possible to test `execVCRMerge` error cases, because it will return an error instead of exiting.

I don't mean think that moving the arg parsing logic into execVcrMerge is necessary; keeping that in RunE makes sense.

Reply:
Currently execVcrMerge does not return any error except at github get pull requests, to test that route to return error, will need to change the mock to make it return error upon some strange parameter?

Reply:
hm, yeah, looking at this again I don't think there's necessarily a reason to add tests for those cases.